src/languages/es-ES.ts

```diff
@@ -1367,6 +1367,15 @@ export default class extends Language {
 					a temporary ban.`,
 			examples: ['@Pete', '@Pete Spamming all channels.', '@Pete Spamming all channels, for 24 hours.']
 		}),
+		COMMAND_DEHOIST_DESCRIPTION: 'Convierta a todos a no izados',
```

Comment:
![image](https://user-images.githubusercontent.com/24852502/71383807-50653980-25de-11ea-89dd-c1841557260f.png)

Reply:
If you **really** want to do Spanish translations, at very least, use this. https://www.wordreference.com/es/translation.asp?tranword=hoist